ci/prow/boskos/set_permissions.sh

```diff
@@ -17,38 +17,44 @@
 set -e
 
 readonly PROJECT=${1:?"First argument must be the boskos project name."}
-readonly PROJECT_OWNERS=("prime-engprod-sea@google.com")
-readonly PROJECT_GROUPS=("knative-productivity-admins@googlegroups.com")
-readonly PROJECT_SAS=(
+
+# APIs, Permissions and accounts to be set.
+readonly RESOURCES=(
+    "roles/owner"
+    "prime-engprod-sea@google.com"
+
+    "roles/editor"
+    "knative-productivity-admins@googlegroups.com"
     "knative-tests@appspot.gserviceaccount.com"
     "prow-job@knative-tests.iam.gserviceaccount.com"
     "prow-job@knative-nightly.iam.gserviceaccount.com"
-    "prow-job@knative-releases.iam.gserviceaccount.com")
-readonly PROJECT_APIS=(
-    "cloudresourcemanager.googleapis.com"
-    "compute.googleapis.com"
-    "container.googleapis.com")
+    "prow-job@knative-releases.iam.gserviceaccount.com"
 
-# Add an owner to the PROJECT
-for owner in ${PROJECT_OWNERS[@]}; do
-  echo "NOTE: Adding owner ${owner}"
-  gcloud projects add-iam-policy-binding ${PROJECT} --member group:${owner} --role roles/owner
-done
+    "roles/viewer"
+    "knative-dev@googlegroups.com"
 
-# Add all GROUPS as editors
-for group in ${PROJECT_GROUPS[@]}; do
-  echo "NOTE: Adding group ${group}"
-  gcloud projects add-iam-policy-binding ${PROJECT} --member group:${group} --role roles/editor
-done
+    # APIs to enable
+    "cloudresourcemanager.googleapis.com"
+    "compute.googleapis.com"
+    "container.googleapis.com"
+)
 
-# Add all service accounts as editors
-for sa in ${PROJECT_SAS[@]}; do
-  echo "NOTE: Adding service account ${sa}"
-  gcloud projects add-iam-policy-binding ${PROJECT} --member serviceAccount:${sa} --role roles/editor
-done
+# Loop through the list of resources and add them.
 
-# Enable APIS
-for api in ${PROJECT_APIS[@]}; do
-  echo "NOTE: Enabling API ${api}"
-  gcloud services enable ${api} --project=${PROJECT}
+role="unknown"
```

Comment:
can we have a comment for this?

Reply:
For the `role` var or for the loop?

Reply:
the role var

Reply:
sorry I should have made it clearer. The reason I asked for a comment was because I'm not sure what will happen if `role` is not initialized as `unknown`. My guess is something bad could happen if role is empty when running `gcloud` command

Reply:
It would fail with "missing role" instead of "unknown role bla". Updated the comment.